packages/plugin-rows/src/editor/index.tsx

```diff
@@ -23,15 +23,18 @@ export const RowsEditor = (
     | undefined
   >(undefined)
 
-  function openMenu(insertIndex: number) {
-    setMenu({
-      index: insertIndex,
-      onClose: pluginState => {
-        rows.insert(insertIndex, pluginState)
-        setMenu(undefined)
-      }
-    })
-  }
+  const openMenu = React.useCallback(
+    (insertIndex: number) => {
+      setMenu({
+        index: insertIndex,
+        onClose: pluginState => {
+          rows.insert(insertIndex, pluginState)
+          setMenu(undefined)
+        }
+      })
+    },
+    [rows]
```

Comment:
I think this is still recreated in every render, since `rows` is our stateType that gets recreated in every render. Did you profile that?

Reply:
profiling results are good: no rerender of RowsEditor when focus or content changes

Reply:
I'm talking specifically about the `React.useCallback` for openMenu. If I'm not mistaken, this does nothing and should work as good without `React.useCallback`

Reply:
good catch, this was due to a prior draft but is unnecessary now, removed the useCallback